package.json

```diff
@@ -12,33 +12,37 @@
   "license": "MIT",
   "private": true,
   "scripts": {
+    "build": "yarn build:clean && yarn build:lib",
+    "build:clean": "yarn rimraf lib",
+    "build:lib": "yarn tsc",
+    "build:watch": "concurrently \"yarn build:watch:lib\"",
+    "build:watch:lib": "yarn tsc --watch",
     "test": "jest",
-    "test:watch": "concurrently \"yarn workspaces run build:watch\" \"jest --watch\"",
-    "build": "yarn wpd build && yarn wd build",
+    "test:watch": "concurrently \"yarn build:watch\" \"jest --watch\"",
     "coverage": "jest --coverage && open coverage/lcov-report/index.html || xdg-open coverage/lcov-report/index.html",
```

Comment:
let's use codecov and also let's add shield for codecov, and you can add also for published version
![image](https://user-images.githubusercontent.com/25525736/187383001-196b731b-1eb0-40c1-92c0-abf14fdc3afb.png)